.env.example

```
@@ -39,6 +39,7 @@ WEB_API_RATE_LIMIT_SESSION_WINDOW_MS=60000 # Time window for session rate limiti
 # Public site key is served to the web app via /config.json (requires secret key to enable)
 TURNSTILE_SITE_KEY=
 TURNSTILE_SECRET_KEY= # Secret key for server-side verification (never expose publicly)
+TURNSTILE_ALLOWED_HOSTNAMES= # Optional comma-separated hostnames allowed in successful Turnstile verification responses
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)